modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevokeWithInvalidClientCredentialsTestCase.java

```diff
@@ -48,6 +50,9 @@
 public class OAuth2TokenRevokeWithInvalidClientCredentialsTestCase extends OAuth2ServiceAbstractIntegrationTest {
 
     private static final String REVOKE_TOKEN_API_ENDPOINT = "https://localhost:9853/oauth2/revoke";
```

Comment:
Can we get these ips & port values using automation.xml file ?
https://localhost:9853/oauth2/token